src/core/config/ip6.h

```diff
@@ -180,4 +180,14 @@
 #define OPENTHREAD_CONFIG_IP6_ALLOW_LOOP_BACK_HOST_DATAGRAMS 1
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_IP6_BR_COUNTERS_ENABLE
+ *
+ * Define as 1 to enable IPv6 Border Routing counters.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_IP6_BR_COUNTERS_ENABLE
+#define OPENTHREAD_CONFIG_IP6_BR_COUNTERS_ENABLE 0
```

Comment:
We can cosnider having this be enabled along with `OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE`?
(we may need to also add `#include "config/border_routing.h"` at the top of this file)
```suggestion
#define OPENTHREAD_CONFIG_IP6_BR_COUNTERS_ENABLE OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
```